src/praisonai-bot/praisonai_bot/gateway/push_delivery.py

```
@@ -24,6 +28,97 @@
+    def load_all(self) -> List[GatewayEvent]:
+        """Return all persisted events (for crash replay on startup)."""
+        with self._lock, closing(self._connect()) as conn:
+            rows = conn.execute(
+                "SELECT payload FROM push_delivery ORDER BY ts ASC"
+            ).fetchall()
+        events: List[GatewayEvent] = []
+        for (payload,) in rows:
+            try:
+                events.append(GatewayEvent.from_dict(json.loads(payload)))
+            except Exception:  # pragma: no cover - defensive
+                continue
```

Ruff (0.16.0)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)